packages/react/src/errorboundary.tsx

```diff
@@ -38,6 +38,8 @@ export type ErrorBoundaryProps = {
   onReset?(error: Error | null, componentStack: string | null, eventId: string | null): void;
   /** Called on componentWillUnmount() */
   onUnmount?(error: Error | null, componentStack: string | null, eventId: string | null): void;
+  /** Called before error is sent to Sentry, allows for you to add tags or context using the scope */
+  beforeSend?(scope: Scope, error: Error | null, componentStack: string | null): void;
```

Comment:
Can we call this method differently, I don't like that we now can confuse global `beforeSend` with this one since it's also works differently.
Alternative: `beforeCapture`

Reply:
Changed to `beforeCapture` 👍